util-taglib/src/com/liferay/taglib/util/AssetCategoryUtil.java

```diff
@@ -86,8 +86,6 @@
 						continue;
 					}
 
-					category = category.toEscapedModel();
```

Comment:
The title is escaped in /html/taglib/ui/asset_categories_selector/page.jsp - line 104 and 144.

AssetCategoryUtil.getCategoryIdsTitles is not used anywhere else.